spopt/region/azp_util.py

```diff
@@ -152,35 +147,31 @@ class AllowMoveAZPMaxPRegions(AllowMoveStrategy):
     spatially extensive attribute.
     """
 
-    def __init__(self, spatially_extensive_attr, threshold,
-                 decorated_strategy):
+    def __init__(self, spatially_extensive_attr, threshold, decorated_strategy):
         """
 
         Parameters
         ----------
-        spatially_extensive_attr : :class:`numpy.ndarray`, default: None
+        spatially_extensive_attr : numpy.ndarray
             See corresponding argument in
-            :meth:`region.max_p_regions.heuristics.MaxPRegionsHeu.fit_from_scipy_sparse_matrix`.
-        threshold : numbers.Real or :class:`numpy.ndarray`
+            ``region.max_p_regions.heuristics.MaxPRegionsHeu.fit_from_scipy_sparse_matrix``.
```

Comment:
just a note to self, really, but we probably want a better description here that doesnt need to references region

Reply:
@knaaptime completely agree. When I started going through everything, I realized the docs will need a lot of love, so I focused on the testing coverage for his PR.

Reply:
lets do it in phases